### PR TITLE
[REFACT]: reorganizing functions

### PR DESCRIPTION
**Pull request by marcosalves-bitg**
This PR is to refact the invoke function and the secretsManager entire folder

### CODE REVIEW

**Comment by Viserion77** [comment]
quebro

File: src/lambda/lambdaService.ts [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
-        ? await SecretManager.getValue({
-          region: lambdaParameters.region,
-          secretId: lambdaParameters.serviceSecretArn
-        })
-        : {}
-
       return executeInvoke({
         ...lambdaParameters,
-        ...accessKey
+        ...SecretManager.getAccessKey(lambdaParameters)
```
Comment (Viserion77) [inline src/lambda/lambdaService.ts:13]:
```suggestion
        ...await SecretManager.getAccessKey(lambdaParameters)
```

**Review by Viserion77 — CHANGES_REQUESTED** [review]

**Review by Viserion77 — APPROVED** [review]
LGTM

**Review by marcelbeskow — APPROVED** [review]
LGTM

**Pull request merged**